MSTuringEmbeddingOnlyScenario.cs

```diff
@@ -14,7 +14,7 @@ internal class MSTuringEmbeddingOnlyScenario : BigANNBinaryEmbeddingOnlyScearioB
         protected override VectorDataType EmbeddingDataType => VectorDataType.Float32;
         protected override DistanceFunction EmbeddingDistanceFunction => DistanceFunction.Euclidean;
         protected override ulong EmbeddingDimensions => 100;
-        protected override int MaxPhysicalPartitionCount => 56;
+        protected override int MaxPhysicalPartitionCount => Convert.ToInt32(this.Configurations["AppSettings:scenario:MaxPhysicalPartitionCount"]);

```

Comment:
Instead of having just 1 scenario taking an input, please :

1. Let scenarios define defaults like we do and pick that if this config is 0.
2. If config is not 0, make change for all scenarios to pick up the config (maybe with a base class?)